src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -350,6 +350,9 @@
   <data name="CommandPalette_SearchBox.PlaceholderText" xml:space="preserve">
     <value>Type a command name...</value>
   </data>
+  <data name="CommandPalette_NoMatchesText.Text" xml:space="preserve">
+    <value>No matches found</value>

```

Comment:
Might want to leave a comment explaining what a match is. I don't want our localizers expanding this as "I could not locate any incendiary devices"

Reply:
Do you think "no matching commands" has the same problem? As Carlos mentioned, that's the text that VsCode uses

Reply:
nope, SGTM